packages/react/src/select/item-text/SelectItemText.tsx

```diff
@@ -59,11 +53,10 @@ export const SelectItemText = React.forwardRef(function SelectItemText(
 ) {
   const { selected, indexRef } = useSelectItemContext();
   const { selectedItemTextRef } = useSelectRootContext();
-  const mergedRef = useForkRef<HTMLElement>(forwardedRef);
 
   return (
     <InnerSelectItemText
-      ref={mergedRef}
+      ref={forwardedRef}
```

Comment:
The forwarded ref may not be memoized

Reply:
Then `useForkRef` would still just re-create a new `mergedRef` because `forwardedRef` would change on every render. So it's just pure overhead to use it here, if I'm not mistaken.